### DISCUSSION TITLE
New placeholders for the punishment system.

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#7991] [Feature Requests]
Please add some placeholders for the /mute, /tempban commands, which will display who issued the punishment, how much time is left before the punishment expires, and the reason for the punishment. And also for the /jail placeholder command, which will display who issued the punishment (because the time before the expiration of the punishment and the reason for the punishment are already there).

**Comment by arimaleroi** [comment]
+1

**Comment by wundertraume** [comment]
+1
